### ISSUE TITLE
Zero exit code when include file does not exist 

### ISSUE THREAD

**Issue by simondean** [Mermade/shins#36]
Hi.  When `shins` terminates due to an include markdown file not existing, `shins` seems to return with a zero exit code.  Expected behaviour would be a non-zero exit code to indicate that something has gone wrong.

**Comment by simondean** [comment]
I've created an example of the issue in https://github.com/simondean/widdershins-issues/tree/master/exit-code

Here's an example of the `shins` output when an include file does not exist, including a printout of `shins`'s exit code: 

``` bash
$ node_modules/.bin/shins index.md -o index.html --inline --attr --customcss
{ Error: widdershins-issues/exit-code/node_modules/shins/source/layouts/layout.ejs:18
    16| <% var language_tabs = current_page.data.language_tabs || []; %>
    17| <% if (current_page.data.includes) { for (var include in current_page.data.includes) { %>
 >> 18|   <% page_content += partial(current_page.data.includes[include]) %>
    19| <% } %>
    20| <!doctype html>
    21| <html lang="en">

ENOENT: no such file or directory, open 'widdershins-issues/exit-code/node_modules/shins/source/includes/_does-not-exist.md'
    at Object.fs.openSync (fs.js:667:18)
    at Object.fs.readFileSync (fs.js:572:33)
    at Object.partial (widdershins-issues/exit-code/node_modules/shins/index.js:69:25)
    at eval (eval at compile (widdershins-issues/exit-code/node_modules/ejs/lib/ejs.js:584:12), <anonymous>:18:24)
    at returnedFn (widdershins-issues/exit-code/node_modules/ejs/lib/ejs.js:615:17)
    at tryHandleCache (widdershins-issues/exit-code/node_modules/ejs/lib/ejs.js:251:36)
    at Object.exports.renderFile (widdershins-issues/exit-code/node_modules/ejs/lib/ejs.js:472:10)
    at widdershins-issues/exit-code/node_modules/shins/index.js:313:13
    at new Promise (<anonymous>)
    at Object.render (widdershins-issues/exit-code/node_modules/shins/index.js:201:28)
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: 'widdershins-issues/exit-code/node_modules/shins/source/layouts/layout.ejs' }
$ echo $?
0
```

**Comment by MikeRalphson** [comment]
Thanks for the report, will take a look.

**Comment by MikeRalphson** [comment]
Should be fixed in `v2.2.1-6`. If `options.cli` is `true` then `shins` will exit with error-code 1 on any non-existing include, script, image etc. If `options.cli` is not truthy then a warning is logged to stderr but rendering will continue.

Thanks again for the prompt.
